lib/packwerk/cli.rb

```diff
@@ -133,17 +133,11 @@ def output_result(result)
       ).returns(FilesForProcessing)
     end
     def fetch_files_to_process(relative_file_paths, ignore_nested_packages)
-      files_for_processing = FilesForProcessing.fetch(
+      FilesForProcessing.fetch(
```

Comment:
I feel like this is only needed if ARGV is empty. If we filter everything out and silently fail I think that's unhelpful to the developer. Can we change it to only abort when we provide files, but none were found.

Reply:
Good call, I changed this to just warn with a `puts`.